lib/hoe/git.rb

```diff
@@ -117,7 +117,7 @@ def define_git_tasks #:nodoc:
       end
 
       task :release_sanity do
-        unless `git status` =~ /^nothing to commit/
+        if `git status --porcelain` =~ /^\?\?/
```

Comment:
Your proposed change doesn't take modified files into account. I think a better check would be that the output is empty:

```ruby
unless `git status --porcelain`.empty?
```

Reply:
Of course.

Reply:
@zenspider bump